scripts/dependency-graph.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Can we add a CI step to generate this automatically and fail if this file is out of date? We do something similar with `yarn.lock` which is very helpful for making sure it doesn't become stale

Reply:
Done